lib/classes/api/commands/class.Admins.php

```diff
@@ -23,7 +23,7 @@ class Admins extends ApiCommand implements ResourceEntity
 	 *
 	 * @access admin
 	 * @throws Exception
-	 * @return array count|list
+	 * @return string
```

Comment:
basically true, a json_encoded array is returned (every api call does) but the extra info should be kept because the phpdoc is parsed for the api-documentation at https://api.froxlor.org/doc/0.10.0/).
So maybe we can agree on

`@return string json-encoded array count|list`?

Reply:
Thats ok for me...